lib/sqlite3.js

```diff
@@ -372,7 +374,7 @@ function _convertBoolean(value) {
     return 1;
   if (booleanFalse.indexOf(value) !== -1)
     return 0;
-  return Error('SQLITE3: Invalid boolean default: ' + value);
+  return Error(g.f('{{SQLITE3}}: Invalid boolean default: %s', value));
```

Comment:
You have to run slt-globalize -e again for intl/en/messages.json to get updated with the brackets change.